archiexpo.es,"HTTPSConnectionPool(host='archiexpo.es', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1124)')))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
archiexpo.it,"HTTPSConnectionPool(host='archiexpo.it', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1124)')))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
arcinfo.ch,"HTTPSConnectionPool(host='arcinfo.ch', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.arcinfo.ch', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1124)')))"
arcotampano.it,"HTTPSConnectionPool(host='arcotampano.it', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbccd0eed90>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='arcotampano.it', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcccaa9310>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
arda.or.th,"HTTPSConnectionPool(host='arda.or.th', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.arda.or.th', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
argep.hu,"HTTPSConnectionPool(host='argep.hu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='argep.hu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
army.mil,"HTTPSConnectionPool(host='army.mil', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbccd842520>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='army.mil', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbccb8c0a30>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
arnes.si,"HTTPSConnectionPool(host='arnes.si', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbccb8b3790>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='arnes.si', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbccb795f70>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
artesanatopassoapassoja.com.br,"HTTPSConnectionPool(host='artesanatopassoapassoja.com.br', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fbcca5a8ca0>, 'Connection to artesanatopassoapassoja.com.br timed out. (connect timeout=2)'))","HTTPConnectionPool(host='artesanatopassoapassoja.com.br', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fbccac6d550>, 'Connection to artesanatopassoapassoja.com.br timed out. (connect timeout=2)'))"
artstation.com,"HTTPSConnectionPool(host='www.artstation.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.artstation.com', port=443): Read timed out. (read timeout=2)"
as-1.co.jp,"HTTPSConnectionPool(host='as-1.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbcce0f7e20>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='as-1.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcca20c8e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
asamblea.gob.ni,"HTTPSConnectionPool(host='asamblea.gob.ni', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbcc9845fd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='asamblea.gob.ni', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbccb4d1340>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
asbeiras.pt,"HTTPSConnectionPool(host='www.asbeiras.pt', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.asbeiras.pt', port=443): Read timed out. (read timeout=2)"
asfi.gob.bo,"HTTPSConnectionPool(host='asfi.gob.bo', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbccca93fa0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='asfi.gob.bo', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcc9d7dfd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
asklaila.com,"HTTPSConnectionPool(host='asklaila.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'asklaila.com' doesn't match '*.asklaila.com'"")))","HTTPSConnectionPool(host='asklaila.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'asklaila.com' doesn't match '*.asklaila.com'"")))"
askoservering.no,"HTTPSConnectionPool(host='askoservering.no', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='askoservering.no', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
asktel.ru,"HTTPSConnectionPool(host='asktel.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='asktel.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
aslimnica.lv,"HTTPSConnectionPool(host='aslimnica.lv', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='aslimnica.lv', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
assabah.com.tn,"HTTPSConnectionPool(host='assabah.com.tn', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbccd922c40>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='assabah.com.tn', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbccb434b80>: Failed to establish a new connection: [Errno 111] Connection refused'))"
assemblee-nationale.fr,"HTTPSConnectionPool(host='assemblee-nationale.fr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbcccb84820>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='assemblee-nationale.fr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcc9ed8d60>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
assorti.lt,"HTTPSConnectionPool(host='assorti.lt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbccc3916d0>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='assorti.lt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbccb8484c0>: Failed to establish a new connection: [Errno 111] Connection refused'))"
astra.co.id,"HTTPSConnectionPool(host='www.astra.co.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.astra.co.id', port=443): Read timed out. (read timeout=2)"
asus.com,"HTTPSConnectionPool(host='asus.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbcce6a9370>: Failed to establish a new connection: [Errno 111] Connection refused'))","('Received response with content-encoding: gzip, but failed to decode it.', error('Error -3 while decompressing data: incorrect header check'))"
at.govt.nz,"HTTPSConnectionPool(host='at.govt.nz', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='at.govt.nz', port=80): Read timed out. (read timeout=2)"
atauni.edu.tr,"HTTPSConnectionPool(host='atauni.edu.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='atauni.edu.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
atelier-povetron.cz,"HTTPSConnectionPool(host='atelier-povetron.cz', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbccc306100>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='atelier-povetron.cz', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcc89a7820>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
atenaeditora.com.br,"HTTPSConnectionPool(host='atenaeditora.com.br', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.atenaeditora.com.br', port=443): Read timed out. (read timeout=2)"
auchan.ro,"HTTPSConnectionPool(host='www.auchan.ro', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.auchan.ro', port=443): Read timed out. (read timeout=2)"
aucklandcouncil.govt.nz,"HTTPSConnectionPool(host='www.aucklandcouncil.govt.nz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.aucklandcouncil.govt.nz', port=443): Read timed out. (read timeout=2)"
aun.edu.eg,"HTTPSConnectionPool(host='aun.edu.eg', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fbcc84961f0>, 'Connection to aun.edu.eg timed out. (connect timeout=2)'))","HTTPConnectionPool(host='aun.edu.eg', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fbcccfa1b50>, 'Connection to aun.edu.eg timed out. (connect timeout=2)'))"
auone.jp,"HTTPSConnectionPool(host='auone.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='auone.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
auti.hr,"HTTPSConnectionPool(host='auti.hr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'auti.hr' doesn't match 'www.auti.hr'"")))","HTTPSConnectionPool(host='auti.hr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'auti.hr' doesn't match 'www.auti.hr'"")))"
auto2000.co.id,"HTTPSConnectionPool(host='auto2000.co.id', port=443): Read timed out.","HTTPSConnectionPool(host='auto2000.co.id', port=443): Read timed out."
autoc-one.jp,"HTTPSConnectionPool(host='autoc-one.jp', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='autoc-one.jp', port=443): Read timed out. (read timeout=2)"
autojerry.fi,"HTTPSConnectionPool(host='autojerry.fi', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='autojerry.fi', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
autojournal.cz,"HTTPSConnectionPool(host='www.autojournal.cz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.autojournal.cz', port=443): Read timed out. (read timeout=2)"
autolux.ro,"HTTPSConnectionPool(host='www.autolux.ro', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.autolux.ro', port=443): Read timed out. (read timeout=2)"
autopower.bg,"HTTPSConnectionPool(host='autopower.bg', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fbcc44f7940>, 'Connection to autopower.bg timed out. (connect timeout=2)'))","HTTPConnectionPool(host='autopower.bg', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fbccbbf0f70>, 'Connection to autopower.bg timed out. (connect timeout=2)'))"
autopurkaamot.com,"HTTPSConnectionPool(host='autopurkaamot.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.autopurkaamot.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
autostation.com,"HTTPSConnectionPool(host='autostation.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'autostation.com' doesn't match 'www.autostation.com'"")))","HTTPSConnectionPool(host='autostation.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'autostation.com' doesn't match 'www.autostation.com'"")))"
autoszektor.hu,"HTTPSConnectionPool(host='autoszektor.hu', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='autoszektor.hu', port=443): Read timed out. (read timeout=2)"
avto-fokus.si,"HTTPSConnectionPool(host='avto-fokus.si', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.avto-fokus.si', port=443): Read timed out. (read timeout=2)"
avto.net,Exceeded 30 redirects.,Exceeded 30 redirects.
avtomobilizem.com,"HTTPSConnectionPool(host='avtomobilizem.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='avtomobilizem.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
awilime.com,"HTTPSConnectionPool(host='awilime.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'awilime.com' doesn't match either of '*.azurewebsites.net', '*.scm.azurewebsites.net', '*.azure-mobile.net', '*.scm.azure-mobile.net', '*.sso.azurewebsites.net'"")))","HTTPSConnectionPool(host='awilime.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'awilime.com' doesn't match either of '*.azurewebsites.net', '*.scm.azurewebsites.net', '*.azure-mobile.net', '*.scm.azure-mobile.net', '*.sso.azurewebsites.net'"")))"
az-gotvia.bg,"HTTPSConnectionPool(host='www.az-gotvia.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.az-gotvia.bg', port=443): Read timed out. (read timeout=2)"
azoo.hr,"HTTPSConnectionPool(host='azoo.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbcc2f30ee0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='azoo.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbccb82edf0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
b2s.co.th,"HTTPSConnectionPool(host='b2s.co.th', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.b2s.co.th', port=443): Read timed out. (read timeout=2)"
baac.or.th,"HTTPSConnectionPool(host='baac.or.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbccc178490>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='baac.or.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcc2f81e80>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
baahball.com,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
babil.com,"HTTPSConnectionPool(host='babil.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='babil.com', port=80): Read timed out. (read timeout=2)"
babnet.net,"HTTPSConnectionPool(host='babnet.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'babnet.net' doesn't match '*.babnet.net'"")))","HTTPSConnectionPool(host='babnet.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'babnet.net' doesn't match '*.babnet.net'"")))"
baby.ru,"HTTPSConnectionPool(host='baby.ru', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbccc2abac0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='baby.ru', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbccc8dab20>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
backapalankavesti.com,"HTTPSConnectionPool(host='backapalankavesti.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='backapalankavesti.com', port=443): Read timed out. (read timeout=2)"
bacninh.gov.vn,"HTTPSConnectionPool(host='bacninh.gov.vn', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbcccf343a0>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='bacninh.gov.vn', port=80): Read timed out. (read timeout=2)"
badenertagblatt.ch,"HTTPSConnectionPool(host='www.badenertagblatt.ch.ch', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbccb7d30a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPSConnectionPool(host='www.badenertagblatt.ch.ch', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbcca128490>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
bahrain.bh,"HTTPSConnectionPool(host='bahrain.bh', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))","HTTPSConnectionPool(host='bahrain.bh', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1124)')))"
balkanspress.com,"HTTPSConnectionPool(host='balkanspress.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbcccf19100>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='balkanspress.com', port=80): Read timed out. (read timeout=2)"
bancoestado.cl,"HTTPSConnectionPool(host='bancoestado.cl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbcc48ffbe0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='bancoestado.cl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcc45671c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
banesco.com,"HTTPSConnectionPool(host='banesco.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbccd0a08b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='banesco.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcc9e42070>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
bangkokassets.com,"HTTPSConnectionPool(host='www.bangkokassets.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.bangkokassets.com', port=443): Read timed out. (read timeout=2)"
bangkokbank.com,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
bangkokhospital.com,"HTTPSConnectionPool(host='www.bangkokhospital.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.bangkokhospital.com', port=443): Read timed out. (read timeout=2)"
banglapedia.org,"HTTPSConnectionPool(host='banglapedia.org', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbcc16a1b50>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='banglapedia.org', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcc31f5280>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
bankalfalah.com,"HTTPSConnectionPool(host='bankalfalah.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbcc3d64760>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='bankalfalah.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcc20f0dc0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
banker.bg,"HTTPSConnectionPool(host='www.banker.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.banker.bg', port=443): Read timed out. (read timeout=2)"
bankleumi.co.il,"HTTPSConnectionPool(host='bankleumi.co.il', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbcc164ff10>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='bankleumi.co.il', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbccc7b4f10>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
bankofgreece.gr,"HTTPSConnectionPool(host='bankofgreece.gr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbccd627580>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='bankofgreece.gr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcc2eecbe0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
banprogrupopromerica.com.ni,"HTTPSConnectionPool(host='banprogrupopromerica.com.ni', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fbcc67c7e20>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='banprogrupopromerica.com.ni', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcc6775640>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
banyuwangikab.go.id,"HTTPSConnectionPool(host='banyuwangikab.go.id', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='banyuwangikab.go.id', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
